### DISCUSSION TITLE
Exam?

### DISCUSSION THREAD

**Discussion by bruluk5w** [sebastian-hofstaetter/teaching#93] [Advanced IR: Exercise Q&A]
Dear Lecture Team,

We have been told that the exam would go online at 12:00 today but there is nothing visible in TUWEL.
Is there a technical hiccup?

Best

Lukas

**Comment by sophiaalthammer** [comment]
Dear Lukas,

I am sorry, we forgot to make the exam visible, it is now visible and online!

Best,
Sophia
